packages/@aws-cdk/aws-codestarnotifications-targets/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+const baseConfig = require('cdk-build-tools/config/eslintrc');
```

Comment:
Can we actually remove the `aws-codestarnotifications-targets` package completely from this PR? It's not required, and it would make this really big PR quite a bit smaller, and easier for me to review. We can bring it back later, in a separate PR.

Reply:
Ok. I will remove.

Reply:
Already removed.